Source/Core/VideoBackends/Vulkan/VideoBackend.h

```diff
@@ -6,11 +6,17 @@
 #include "Common/Common.h"
 #include "VideoCommon/VideoBackendBase.h"
 
+namespace Core
+{
+class System;
+}
+
 namespace Vulkan
 {
 class VideoBackend : public VideoBackendBase
 {
 public:
+  VideoBackend(Core::System& system);
```

Comment:
```suggestion
  using VideoBackendBase::VideoBackendBase;
```

Derived constructors that want to just use the base class constructors can be implemented more simply like this.
It looks like most/all of the `VideoBackend` and `VertexManager` classes are like this.

That also makes the forward declaration of `System` above unnecessary. I guess it already kinda was considering the base class header needs and declares `System`.

Reply:
I now pass `System` into the `Initialize` and `Shutdown` functions instead, but I'll keep that in mind in the future.